### PR TITLE
Added People List not working yet

### PR DESCRIPTION
**Pull request by MirekSobota**
People list with a working search and mobile max 320px. Pagination doesn't work yet.

Edit:
Added more breakpoints.

Please let me know what do you think!

### CODE REVIEW

File: src/features/PeopleList/index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { useDispatch, useSelector } from "react-redux";
+import debounce from "lodash/debounce";
```
Comment (PawelOwiesek) [inline src/features/PeopleList/index.js:2]: This is not correct import

Comment (MirekSobota) [reply src/features/PeopleList/index.js:2]: Why?

Comment (PawelOwiesek) [reply src/features/PeopleList/index.js:2]: lodash/debounce?

Comment (MirekSobota) [reply src/features/PeopleList/index.js:2]: It is ok i used this one

File: src/common/TileList/TilePeople/Image/style.js
Comment (DorotaZielinska) [inline src/common/TileList/TilePeople/Image/style.js:1]: Only name of file from style.js to styled.js and import will change name as well. The rest is great, congratulationa :D

**Review by DorotaZielinska — APPROVED** [review]

**Pull request merged**